testsuite/MDAnalysisTests/__init__.py

```diff
@@ -101,7 +101,7 @@
 
 import os
 import sys
-
+os.environ['DUECREDIT_ENABLE']='yes'
```

Comment:
Please add a comment that this is for testing duecredit in utils/test_duecredits.py